### ISSUE TITLE
Missing documentation part for Page Types since v12

### ISSUE THREAD

**Issue by nhovratov** [TYPO3-Documentation/TYPO3CMS-Reference-CoreApi#5050]
### Current behavior

It's not documented how to create the showitems string for Page Types in "Create new Page Type" documentation.

### Expected behavior/output

The part should be brought back how it was in v11.

### Links

v12: https://docs.typo3.org/m/typo3/reference-coreapi/12.4/en-us/ApiOverview/PageTypes/CreateNewPageType.html
v11: https://docs.typo3.org/m/typo3/reference-coreapi/11.5/en-us/ApiOverview/PageTypes/CreateNewPageType.html
Forge issue: https://forge.typo3.org/issues/105605

### TYPO3 versions

main, v12

**Comment by linawolf** [comment]
Yep no one was willing to keep the example updated and I was told that it is not public API anyway so I removed the example.

If you are willing to attak the topic again please go ahead.

**Comment by linawolf** [comment]
Solved with #5341
